README.md

```diff
@@ -1,316 +1,588 @@
 ## [HTML](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics)
-- [ ] HTML main tags 
-  - [ ] `<html>` [Link](https://www.w3docs.com/learn-html/html-html-tag.html)
-  - [ ] `<head>` [Link](https://www.w3docs.com/learn-html/html-head-tag.html)
-  - [ ] `<body>` [Link](https://www.w3docs.com/learn-html/html-body-tag.html)
-  - [ ] `<script>` [Link](https://www.w3docs.com/learn-html/html-script-tag.html)
-  - [ ] `<style>` [Link](https://www.w3docs.com/learn-html/html-style-tag.html)
-  - [ ] `<link>` [Link](https://www.w3docs.com/learn-html/html-link-tag.html)
-  - [ ] `<meta>` [Link](https://www.w3docs.com/learn-html/html-meta-tag.html)
-  
+
+- [ ] HTML main tags [Playlist](https://www.youtube.com/playlist?list=PLblA84xge2_xNtaFnZhefjFbnDrpySKD3)
+
```

Comment:
Looks like there are formatting issues. Can you remove the extra spaces?

Reply:
Actually, prettier added those white spaces. Do you use some formating tool to format the markdown?